app/templates/partials/gtm.html

```diff
@@ -1,18 +1,12 @@
 <!-- Google Tag Manager -->
-<script nonce="{{ cspNonce }}">
```

Comment:
This needs to stay here. I think the security policy starts to error on this from what I can see in the chrome console